### PR TITLE
chore(peerDependencies): @spinnaker/pluginsdk-peerdeps - Synchronize peerdependencies with root project

### PR DESCRIPTION
**Pull request by abannachZen**
While scaffolding a new deck plugin I noticed that the peerDependencies provided by `@spinnaker/pluginsdk-peerdeps` were out of date. Per the readme, I ran `yarn sync` to synchronize the peerdependencies with the root project. 

I also noticed the `README.md` had an outdated yarn script so I updated that as well.

### CODE REVIEW

**Comment by abannachZen** [comment]
> The following commits need their title changed:
> 
> * [30358bb](https://github.com/spinnaker/deck/commit/30358bb41f3667b50a004ebb6549f67ab3ee6904): Synchronize peerdependencies with root by running 'yarn sync'. Update readme to have proper commands
> 
> Please format your commit title into the form:
> 
> `<type>(<scope>): <subject>`, e.g. `fix(kubernetes): address NPE in status check`
> 
> This allows us to easily generate changelogs & determine semantic version numbers when cutting releases. You can read more about [commit conventions](https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions) here.

Updated the commit message to follow standards.

**Review by mattgogerly — APPROVED** [review]
LGTM

**Comment by abannachZen** [comment]
@mattgogerly are you able to approve running workflows for this PR? I'm a first time contributor to this repo. Thanks!

**Comment by mattgogerly** [comment]
> @mattgogerly are you able to approve running workflows for this PR? I'm a first time contributor to this repo. Thanks!

Unfortunately no, I don't have write access.

paging @caseyhebebrand @christopherthielen

File: packages/pluginsdk-peerdeps/README.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ These fields inform a plugin which versions of packages it should install using
 
 # Updating this package
 
-The source of truth for package verions comes from deck (`../../package.json`) for most packages. This package has a few
-script that help keep the `peerDependencies` up to date.
+The source of truth for package verions comes from deck (`<project-root>/package.json`) for most packages. This package has a few
```
Comment (dbyron-sf) [inline packages/pluginsdk-peerdeps/README.md:11]:
```suggestion
The source of truth for package versions comes from deck (`<project-root>/package.json`) for most packages. This package has a few
```

**Review by dbyron-sf — APPROVED** [review]
not an expert in deck dependency mgmt, but lgtm.

**Comment by abannachZen** [comment]
@dbyron-sf It feels like this type of PR could be automated via a github action of some sort. Like an action to watch changes to the `<project-root>/package.json` on a branch and if it changes run the sync operation as well. This way it can never get outdated.

**Comment by dbyron-sf** [comment]
@abannachZen I bet you're right, though I'm not sure I'm the right person to do it.  Any chance the way forward is clear to you by looking at https://github.com/spinnaker/deck/tree/master/.github/workflows?  Or perhaps @link108 knows?

**Pull request merged**